code/modules/unit_tests/mouse_bite_cable.dm

```diff
@@ -2,19 +2,53 @@
-	biter.chew_probability = 100 // Make sure the mouse bites the cable!
-	wire.powernet = new /datum/powernet() // Make sure the cable has a powernet.
-	wire.powernet.avail = 100000 // Make sure the powernet has a good amount of power! handle_automated_action checks to see if there is ANY power in the powernet and passes fine, but better safe than sorry in this instance.
+	// Make sure the cable has a powernet.
+	wire.powernet = new()
+	// Make sure the powernet has a good amount of power!
+	// mice bites check if there is ANY power in the powernet  and passes fine, but better safe than sorry
+	wire.powernet.avail = 100000
 
 	var/turf/open/floor/stage = get_turf(wire)
-	stage.underfloor_accessibility = UNDERFLOOR_INTERACTABLE // the unit tests room has normal flooring so let's just make it be interactable for the sake of this test
-
+	// the unit tests room has normal flooring so let's just make it be interactable for the sake of this test
+	stage.underfloor_accessibility = UNDERFLOOR_INTERACTABLE
+	// relocate the rat
 	biter.forceMove(stage)
-	biter.handle_automated_action() // it's not so automated since we're forcing it and doing everything we can to ensure that mice fucking bites that wire but potato potato
 
-	TEST_ASSERT(QDELETED(biter), "Mouse did not die after biting a powered cable.") // we qdel the mouse mob on death
-	TEST_ASSERT(QDELETED(wire), "Cable was not deleted after being bitten by a mouse.")
+	// Ai controlling processes expect a delta_time, supply a real-fake dt
+	var/fake_dt = SSai_controllers.wait * 0.1
+	// Select behavior - this will queue finding the cable
+	biter.ai_controller.SelectBehaviors(fake_dt)
+	// Process behavior - this will execute the "locate the cable" behavior
+	biter.ai_controller.process(fake_dt)
+	// Check that the cable was found
+	TEST_ASSERT(biter.ai_controller.blackboard[BB_CURRENT_HUNTING_TARGET] == WEAKREF(wire), "Mouse, after executing find, did not set the cable as a target.")
+	// Select behavior - this will queue hunting
+	biter.ai_controller.SelectBehaviors(fake_dt)
+	// Process behavior - this will execute the hunt for the cable and cause a bite (as we're in the min range)
+	biter.ai_controller.process(fake_dt)
+	// Check that the cable was removed, as it was hunted correctly
+	TEST_ASSERT_NULL(biter.ai_controller.blackboard[BB_CURRENT_HUNTING_TARGET], "Mouse, after executing hunt, did not clear their target blackboard.")
+
+	// Now check that the bite went through - remember we qdel mice on death
+	TEST_ASSERT(QDELETED(biter), "Mouse, did not die after biting a powered cable.")
+	TEST_ASSERT(QDELETED(wire), "Cable, was not deleted after being bitten by a mouse.")
+
+	// reset the floor to its original state, to be nice to other tests in case that matters
+	stage.underfloor_accessibility = initial(stage.underfloor_accessibility)
+
+
+// Dummy mouse that is guaranteed to bite the cable beneath it when told.
+/mob/living/basic/mouse/cable_lover
+	cable_zap_prob = 100
```

Comment:
```suggestion
// Dummy mouse that is guaranteed to bite the cable and die biting the cable beneath it when told.
/mob/living/basic/mouse/cable_lover
	cable_zap_prob = 100
```

Since you're setting cable_zap_prob to 100, and you doc it as `Prob that we get zapped and die to a powered cable`, it's probably important to note that we max out the probability of the mouse dying after they bite the cable so the QDELETED check for `biter` (line 36) should always ASSERT correctly.